### DISCUSSION TITLE
Évaluation de la qualité de la colorisation

### DISCUSSION THREAD

**Discussion by LucaRom** [IGNF/myria3d#92] [Q&A] [answered]
Bonjour,

Merci pour la librairie. Avez-vous une façon d'évaluer la qualité de la colorisation et l'impact qu'elle peut avoir sur la segmentation finale? Par exemple, si la colorization de certains LAZ est décalée par une orthorectification imparfaite, quel sera l'impact sur l'entraînement.

Merci!

**Comment by leavauchier** [comment, marked as answer]
Bonjour LucaRom, 

Non, nous n'avons pas prévu d'évaluer la qualité de la colorisation car le projet a été développé dans un cadre dans lequel nous avons accès à une unique source de colorisation (https://geoservices.ign.fr/bdortho), et nous avons considéré que le calage était suffisemment bon (ou que le travail de recalage était en dehors de notre champ d'action)
Je suppose qu'il y aura effectivement un impact si la colorisation est décalée. ou si la source de colorisation n'est pas la même entre apprentissage et inférence.

Dans tous le cas, si vous vous lancez dans ce type d'évaluation, nous serons intéressés par vos potentielles conclusions

**Reply by LucaRom** [reply]
Merci pour la réponse et les explications!
